app/views/user_notifications/digest.html.erb

```diff
@@ -156,24 +156,19 @@
 </table>
 <%- end %>
 
-<table class="digest-topic-stats with-dir" style="padding:0;vertical-align:top;width:100%; margin-top:20px;">
+<table class="digest-topic-stats with-dir" style="padding:0;vertical-align:top;width:100%; padding-top:20px;">
```

Comment:
I wonder whether this change is necessary... older clients can be very very finicky with support padding vs margin on a `table` element. If it's not strictly necessary, it would be good to avoid making this change IMO.

Reply:
It is necessary!

with margin we can't tweak the background colors as we need to:
<img width="960" alt="Screen Shot 2022-04-05 at 12 16 37 PM" src="https://user-images.githubusercontent.com/50783505/161813643-7f3f2f6e-69c3-46bf-a947-5c1af7142df6.png">

with padding: 
<img width="990" alt="Screen Shot 2022-04-05 at 12 17 43 PM" src="https://user-images.githubusercontent.com/50783505/161813607-1f3975ba-9a6d-4f34-85a3-d82b85e72d6f.png">